source/gui/_addonStoreGui/controls/storeDialog.py

```diff
@@ -281,10 +273,15 @@ def _titleText(self) -> str:
 
 	@property
 	def _listLabelText(self) -> str:
-		return f"{self._statusFilterKey.displayString}"
+		return pgettext(
+			"addonStore",
+			# Translators: The label of the add-on list in the add-on store; {category} is replaced by the selected
+			# tab's name.
+			"{category}:",
+		).format(category=self._statusFilterKey.displayStringWithAccelerator)
 
 	def _setListLabels(self):
-		self.listLabel.SetLabelText(self._listLabelText)
+		self.listLabel.SetLabel(self._listLabelText)

```

Comment:
is there a reason why this was changed? We use `SetLabelText` elsewhere so perhaps those instances should be changed too

Reply:
`SetLabelText` seems to display the raw text, i.e. the "&" character appears in the label's text (e.g. "Installed &add-ons") and the accelerator key is not created. On the contrary, `SetLabel` converts "&" character to an accelerator key and is removed from the displayed text (e.g. "Installed add-ons").

Reply:
Ah - it doesn't look this is will be an issue for the other usages.